package.json

```diff
@@ -79,6 +79,7 @@
     "passport-oauth2": "^1.4.0",
     "path-to-regexp": "^2.4.0",
     "prop-types": "^15.7.2",
+    "qs": "^6.5.2",
```

Comment:
Adding explicit dependency here (although it's already transitively used).

We currently use `bodyParser.urlencoded({ extended: **false** })` so posted forms are parsed with the query-string library rather than qs.  This means that tricks like form input arrays to group inputs doesn't work out of the box.  I've opted for using qs directly where I've needed it since changing to `extended: true` across the board seems quite a dangerous change.